gems/pending/VixDiskLib/VixDiskLib.rb

```diff
@@ -96,7 +96,7 @@ def self.setup_env
     vars_to_clear.each do |key|
       my_env.delete(key)
     end
-    my_env["LD_LIBRARY_PATH"] = my_env["LD_LIBRARY_PATH"] + ":" + VIXDISKLIB_PATH
+    my_env["LD_LIBRARY_PATH"] = my_env["LD_LIBRARY_PATH"].to_s + ":" + VIXDISKLIB_PATH
```

Comment:
Yes.  It still works.  Each colon-separated path is inspected to find the
libraries.  It obviously won't find anything in the first path.

On Tue, Aug 18, 2015 at 3:49 PM, Matthew Draper <notifications@github.com>
wrote:

> In gems/pending/VixDiskLib/VixDiskLib.rb
> <https://github.com/ManageIQ/manageiq/pull/3932#discussion_r37344499>:
>
> > @@ -96,7 +96,7 @@ def self.setup_env
> >      vars_to_clear.each do |key|
> >        my_env.delete(key)
> >      end
> > -    my_env["LD_LIBRARY_PATH"] = my_env["LD_LIBRARY_PATH"] + ":" + VIXDISKLIB_PATH
> > +    my_env["LD_LIBRARY_PATH"] = my_env["LD_LIBRARY_PATH"].to_s + ":" + VIXDISKLIB_PATH
>
> Is it okay to leave a blank value at the start of the list?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/3932/files#r37344499>.
>



-- 

-- 
Jerry Keselman
Red Hat Virtualization R&Djerryk@redhat.com | p. 212 530 7873